### ISSUE TITLE
NPM 5.7.1+ `npm ci` vs. `npm install --no-save` for dist/prod (`npm run package` electron-builder, or `npm run start` during last-stage testing)

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#114]
Also important: which NodeJS and NPM version used when lead dev creates DMG etc. installer

**Comment by danielweck** [comment]
Note that NPM `5.8.0` seems to have fixed the `5.7.1` bug which was preventing a reliable upgrade to the new `npm ci` command.
Tested with:
```
sudo npm cache clean --force
sudo npm --global update --unsafe-perm=true --allow-root

# sudo npm install --global npm@5.6.0  --unsafe-perm=true --allow-root
```
(last line to optionally revert to last-know-working version of NPM)

**Comment by danielweck** [comment]
Allow me to close this issue (started by me) as it was just a question which we have since been discussing. Basically we are well-aware that some NPM versions are problematic with the `node_module` package hierarchy, especially with `github:` dependencies (even with `semver` tagging).
